web/js/containers/animation-widget.js

```diff
@@ -1,23 +1,23 @@
 import React from 'react';
 import { connect } from 'react-redux';
 import {
-  without as lodashWithout,
   find as lodashFind,
   get as lodashGet
 } from 'lodash';
+import googleTagManager from 'googleTagManager';
```

Comment:
```suggestion
```

Reply:
Looks like we are still using this to track gif creation.  I just moved it up in the list of imports by a few lines.